docs/handbook/strategy.md

```diff
@@ -66,6 +66,67 @@ What we will build:
 
 Making money:
 
-* We will focus on small customers first. This is because we will optimize for learning. There are more of them and they will be faster to learn from. Product analytics tools often charge 100 person startups $20-70K/year.
-* We can undercut SAAS product analytics tools because we have no hosting costs.
-* Over time, we will put more work into our innovation, and the biggest and brightest companies will realize that we are the market leader.
+We'll have three routes to generating revenue
+
+* Set up a self-serve hosted version
+* Build some of the growth/ultimate plan features like experimentation
+* Support self-hosted
+
+## Hiring
+
+Now that we've raised our seed, we have the money to comfortably hire about 8 remote people and have 3 years of runway, taking us to the end of Q2 2023.
+
+A couple of "rules":
+* Tim and James would like to remain [working managers](https://m.signalvnoise.com/this-is-why-we-have-working-managers-at-basecamp-and-why-microsoft-and-apple-stumbled-when-they/).
+  * That means hiring [managers of one](https://signalvnoise.com/posts/1430-hire-managers-of-one).
+* We are too small for tiered management, and will be until after we raise a Series A.
+* We don't _have_ to hire all the people we have money for. In fact, we should find ways of achieving our roadmap with the fewest people possible.
+
+At the moment, our team is heavily weighted towards frontend/backend development. We're currently lacking:
+* Deep experience with DevOps and the various cloud providers
+  * We need this because we want to be able to deploy PostHog on as many platforms as possible + support
+* Experience scaling systems to 100s of requests/sec
+* Experience scaling Postgres to terabytes
+* Great visual/UX designer
+* Marketing/growth
+
+We could also use more firepower to deliver features faster, though we should be careful with hiring more generalists early on. We could end up needing too much co-ordination. The priority should be a devops/backend scaling generalist.
+
+### The plan:
```

Comment:
Agreed on the generalist point.

A few arguments, feel free to disagree.

1 - We should hire a scalability / dev ops engineer now.

* They can work on scalability so we can get higher volume users live. We are already having users requiring this, and these users are the ones most likely to need managed deployment.
* They can make sure the platform works in lots of different places so we get as much usage as possible.
* They can work on making sure it's easy to deploy the paid version of the software by upgrading from the free version so we can generate revenue. We can't do this until we build more paid features, however the most key one is likely to be managed deployment.

2 - We should hire a visual / UX person now.

* They can work in parallel to everything else. It'll be harder to improve our design later on. They could work on (i) website and branding (ii) UX of the platform (iii) scoping new stuff.

3 - We could hire a full stack person once Phase 2 (dev features) of the roadmap is scoped, which could be very soon, but this is probably going to lead to too many generalists

* This could mean we get the basics of Phase 2 done sooner, so it's easier to get a growth engineer in and to focus on growth / set up. HOWEVER, the downside is coordination problems - would these outweigh this benefit?
* Everything in our todo list is a full stack task, so you'd probably have too much overlap to bring them into this list:

![Screenshot 2020-04-26 at 19 44 31](https://user-images.githubusercontent.com/47497682/80316838-0f816980-87f8-11ea-90d2-8b88630d389c.png)

4 - no need to hire a marketing or growth engineer yet.

* We should do more validation of the channels that work to get more usage. Content will probably work, but we need something more repeatable.
* A growth engineer would have a lot of "sign up process" optimization to do. If our product is dramatically changing, this feels like it'd not be a good full time role yet since a lot of the work would get thrown away repeatedly.

Reply:
1. Done, and added clarification on scale.
2. Added as per discussion below -- we'll do contractor now then see if we want to go full time
3. Agree, devops will be first hire then will move to more generalist.
4. Agree, removed from plan and added note.